src/test/java/com/opentok/test/Helpers.java

```diff
@@ -52,14 +57,32 @@ public static boolean verifyTokenSignature(String token, String apiSecret) throw
         return (signature.equals(signData(decodedParts[1], apiSecret)));
     }
 
-    public static void verifyPartnerAuth(int apiKey, String apiSecret) {
-        verify(RequestPatternBuilder.allRequests()
-                .withHeader("X-TB-PARTNER-AUTH", matching(apiKey + ":" + apiSecret)));
+
+    public static Map<String, Object> decodeToken(String token, Integer apiKey, String apiSecret)
+            throws UnsupportedEncodingException, InvalidJwtException {
+        return getClaims(token, apiKey, apiSecret);
+    }
+
+    public static boolean verifyTokenSignature(String token, Integer apiKey, String apiSecret) {
+
+        try {
+            getClaims(token, apiKey, apiSecret);
```

Comment:
It's a bit weird that the `getClaims()` return value is completely ignored. It's a code smell. Can we figure out how to separate pulling the claims out from verifying the token signature?